### ISSUE TITLE
Non-Default database does not work with PostgreSQL

### ISSUE THREAD

**Issue by mreed19** [koxudaxi/local-data-api#44]
**Describe the bug**
Using a database other than the default database (`postgres`) does not work.

**To Reproduce**
1. Start Postgres Docker Compose

```
docker-compose -f docker-compose-postgres.yml up
```

2. Run AWS rds-data execute statement `SELECT current_database();`

```
aws --endpoint-url http://localhost:8080 rds-data execute-statement --database 'test' --resource-arn "arn:aws:rds:us-east-1:123456789012:cluster:dummy" --secret-arn "arn:aws:secretsmanager:us-east-1:123456789012:secret:dummy" --sql "SELECT current_database();"
```

**Expected behavior**
Result should be:

```
{
    "numberOfRecordsUpdated": 0,
    "records": [
        [
            {
                "stringValue": "test"
            }
        ]
    ]
}
```

**Actual Behavior**
Actual results:

```
{
    "numberOfRecordsUpdated": 0,
    "records": [
        [
            {
                "stringValue": "postgres"
            }
        ]
    ]
}
```

**Desktop (please complete the following information):**
 - OS: macOS Catalina
 - Version: 10.15.2

**Additional context**
I believe this is due to `resource.use_database` executing a `use {database_name}` statement. This works for MySQL but not for PostgreSQL where a connection to the database needs to be established and cannot be changed using SQL.

**Comment by koxudaxi** [comment]
@mreed19 
Thank you for your report.
I have released a fixed version as version 0.4.5.

**Comment by mreed19** [comment]
That fixed it! Thanks so much for resolving this so quickly.
